source/reference/minio-mc-deprecated.rst

```diff
@@ -51,6 +51,22 @@ Table of Deprecated Commands
      - :mc-cmd:`mc ilm rule rm`
      - mc RELEASE.2022-12-24T15-21-38Z
 
+   * - ``mc quota``
```

Comment:
Not entirely sure what to do about `mc quota` itself. There's important information there about how bucket quota works (and why it's not what people think it is). So I don't want to just drop its page.

Maybe put that information into the deprecated `mc quota set` page instead? 🤔

Reply:
Ok, did the above so there's no `mc quota` page under deprecated.